src/core/Akka.TestKit.Tests/TestKitBaseTests/DilatedTests.cs

```diff
@@ -26,47 +28,49 @@ public DilatedTests()
         [Fact]
         public void Dilates_correctly_using_timeFactor()
         {
-            Assert.Equal(Dilated(TimeSpan.FromSeconds(1)), TimeSpan.FromSeconds(4));
+            Assert.Equal(Dilated(TimeSpan.FromMilliseconds(Timeout)), TimeSpan.FromMilliseconds(ExpectedTimeout));
         }
 
         [Fact]
         public void AwaitCondition_should_dilate_timeout()
         {
-            var before = Now;
-            Intercept(() => AwaitCondition(() => false, TimeSpan.FromMilliseconds(Timeout)));
-            var after = Now;
-            var diff = (after - before).TotalMilliseconds;
-            Assert.True(Math.Abs(diff - ExpectedTimeout) <= DiffDelta);
```

Comment:
Timeout should always be greater than the dilated timeout time, not plus or minus a certain margin.